e2e-tests/trade-finance-manager/cypress/fixtures/deal-AIN.js

```diff
@@ -194,7 +194,7 @@ const MOCK_DEAL = {
       'coverEndDate-month': dateConstants.oneMonthMonth,
       'coverEndDate-year': dateConstants.oneMonthYear,
       issuedDate: `${dateConstants.twoYearsAgoUnix}000`,
-      requestedCoverStartDate: '1606900616652',
+      requestedCoverStartDate: `${dateConstants.twoYearsAgoUnix}000`,
```

Comment:
What are the triple zeros for? Otherwise LGTM

Reply:
milliseconds in the epoch timestamp.  Will break without ms